addons/mail/static/src/activity/activity_markasdone_popover.js

```diff
@@ -0,0 +1,57 @@
+/** @odoo-module */
+
+import { useMessaging } from "@mail/messaging_hook";
+import { Component, onMounted, useExternalListener, useRef } from "@odoo/owl";
+import { useService } from "@web/core/utils/hooks";
+
+export class ActivityMarkAsDone extends Component {
+    static template = "mail.activity_mark_as_done";
+    static props = ["activity", "close?", "hasHeader?", "headerText?", "reload?"];
+    static defaultProps = {
+        hasHeader: false,
+    };
```

Comment:
Decision was to keep current linter, and therefore replace `static` attributes by ES6 equivalent `Object.assign(ActivityMarkAsDone, {})`

Reply:
yes I forgot to convert them back